SPEC/FILES.md

```diff
@@ -884,6 +958,43 @@ console.log(stats)
 // }
 ```
 
+#### `files.touch`
+
+> Update the mtime of a file or directory
+
+##### `ipfs.files.touch(path, [options])`
+
+Where:
+
+- `path` is the path to the file or directory to update. It might be:
+  - An existing MFS path to a file or directory (e.g. `/my-dir/a.txt`)
+  - An IPFS path (e.g. `/ipfs/QmWGeRAEgtsHW3ec7U4qW2CyVy7eA2mFRVbk1nb24jFyks`)
+  - A [CID][cid] instance (e.g. `new CID('QmWGeRAEgtsHW3ec7U4qW2CyVy7eA2mFRVbk1nb24jFyks')`)
+- `options` is an optional Object that might contain the following keys:
+  - `mtime` A Date object, an object with `{ sec, nsecs }` properties or the output of `process.hrtime()` or `process.hrtime.bigint()`  (default: now)
```

Comment:
```suggestion
  - `mtime` A ` Date` object or an object with `{ sec, nsecs }` properties or the output of `process.hrtime()` or `process.hrtime.bigint()`  (default: now)
```

Reply:
TBH we can't use `process.hrtime.bigint()` until https://github.com/ipfs/aegir/issues/487 is resolved.

```suggestion
  - `mtime` Either a ` Date` object, an object with `{ sec, nsecs }` properties or the output of `process.hrtime()`  (default: now)
```